parsers/NORDPOOL.py

```diff
@@ -0,0 +1,194 @@
+from dataclasses import dataclass
+from datetime import datetime, timedelta, timezone
+from enum import Enum
+from logging import Logger, getLogger
+
+from requests import Response, Session
+
+from electricitymap.contrib.lib.models.event_lists import PriceList
+from electricitymap.contrib.lib.types import ZoneKey
+
+from .lib.utils import get_token
+
+NORDPOOL_BASE_URL = "https://data-api.nordpoolgroup.com/api/v2/"
+
+
+@dataclass
+class NordpoolToken:
+    token: str
```

Comment:
@VIKTORVAV99 I changed some behavior here.
I removed the possibility of having a None str or a None expiration date. If a token is created then it should always have both otherwise a token cannot be created.
This is a better practice than before by bundling the two and making sure they are always attached to each other.